### PR TITLE
review api

### PR DESCRIPTION
**Pull request by merryjoseph**

### CODE REVIEW

**Review by AvinSJ — APPROVED** [review]

File: src/main/java/com/sayone/ebazzar/common/RestResources.java [hunk 1]
```diff
@@ -20,4 +20,23 @@
     String PRODUCT_POST_SAVE = "/create";
     String PRODUCT_PUT_UPDATE = "/update/{id}";
     String PRODUCT_GET_SEARCH = "/search";
+
+    /*
+     * API Endpoints for Review related operations
+     * */
+//    String REVIEW_ROOT = "/customer";
```
Comment (issac-sayone) [inline src/main/java/com/sayone/ebazzar/common/RestResources.java]: why is it commented ? if it is not need, don't commit else give proper doc comments.

File: src/main/java/com/sayone/ebazzar/controller/ReviewController.java [hunk 2]
```diff
@@ -1,7 +1,64 @@
...
+
+    @PostMapping(
+            consumes = {MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_XML_VALUE},
+            produces = {MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_XML_VALUE})
+    public ReviewDto createReview(
+            @RequestBody ReviewRequestModel reviewRequestModel)  {
+
+        ReviewDto returnValue = new ReviewDto();
+
+        if (reviewRequestModel.getProductId() == 0)
```
Comment (issac-sayone) [inline src/main/java/com/sayone/ebazzar/controller/ReviewController.java]: move the business logic to service classes. controllers should be short and clean.

File: src/main/java/com/sayone/ebazzar/entity/AddressEntity.java [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 import javax.persistence.*;
 
-@Entity(name="addressEntity")
+@Entity(name="address")
```
Comment (issac-sayone) [inline src/main/java/com/sayone/ebazzar/entity/AddressEntity.java]: take a look at table naming conventions.

File: src/main/java/com/sayone/ebazzar/entity/AddressEntity.java [hunk 4]
```diff
@@ -3,7 +3,7 @@
...
 public class AddressEntity {
```
Comment (issac-sayone) [inline src/main/java/com/sayone/ebazzar/entity/AddressEntity.java:11]: All tables should have created/modified timestamp. Create a base table and use it.

File: src/main/java/com/sayone/ebazzar/entity/CartEntity.java [hunk 5]
```diff
@@ -6,7 +6,7 @@
 import javax.persistence.Table;
 
 @Entity
-@Table
+@Table(name = "cart")
 public class CartEntity {
```
Comment (issac-sayone) [inline src/main/java/com/sayone/ebazzar/entity/CartEntity.java:8]: created/modified timestamp

File: src/main/java/com/sayone/ebazzar/entity/OrderEntity.java [hunk 6]
```diff
@@ -39,4 +71,46 @@ public int getOrderAmount() {
...
+
+    public UserEntity getUserEntity() {
+        return userEntity;
+    }
+
+    public void setUserEntity(UserEntity userEntity) {
+        this.userEntity = userEntity;
+    }
+
+//    public List<ProductEntity> getProductEntityList() {
```
Comment (issac-sayone) [inline src/main/java/com/sayone/ebazzar/entity/OrderEntity.java]: remove unwanted lines of code

**Review by issac-sayone — CHANGES_REQUESTED** [review]

**Pull request closed without merging**